frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -595,18 +631,38 @@ function EditAnsibleJobModal(props: {
             onClose={() => props.setAnsibleJob()}
         >
             <AcmForm>
+                <span style={{ display: 'inline-flex' }}>
+                    <Radio
+                        name={'job-template'}
+                        id={'job-template'}
+                        label={t('Job template')}
+                        isChecked={filterForJobTemplates}
+                        className={classes.radio}
```

Comment:
Please use a `<FormGroup>` with `isInline` property instead of introducing custom CSS.

Reply:
It seems like `isInline` is not a prop on `<FormGroup>` or on `<Radio>` So I've applied a `<Split>` instead. Please let me know if I'm mistaken!

Reply:
I meant `<FormGroup>` from `@patternfly/react-core`, not `@material-ui/core`.